### DISCUSSION TITLE
ITMS-90668 Invalid Bundle Executable

### DISCUSSION THREAD

**Discussion by omardajani1** [codemagic-ci-cd/codemagic-docs#1134] [Q&A] [answered]
Hello, I have a flutter app that i will publish to the app store. I am facing an error after i upload the ipa file to app store connect.
The error says: 

### ITMS-90668: Invalid Bundle Executable - The executable file 'Runner.app/Frameworks/Flutter.framework/Flutter' contains incomplete bitcode. To compile binaries with complete bitcode, open Xcode and choose Archive in the Product menu.

I have tried setting this in the podfile: 
**`config.build_settings['ENABLE_BITCODE'] = 'NO'`** 
but it still gives the error when i upload it to appstore connect. 

Some solutions suggest opening the build settings in xcode and manually setting the ENABLE_BITCODE to NO from there.
This is not possible to try for me since i cannot access xcode on the build machine provided by codemagic.
I've been dealing with this issue for a couple of days now and nothing seems to resolve it.
If anyone has faced a similar issue while trying to upload a flutter app built for ios on codemagic please help.

**Comment by himesh-cm** [comment, marked as answer]
Hi, @keyboard-warrior1. Do you have a local Xcode environment?

**Reply by himesh-cm** [reply]
Hi @keyboard-warrior1, can you please confirm if you still face this issue?

**Reply by omardajani1** [reply]
No, the issue was resolved when I changed the build script 
from 
`flutter build ios - -release`
to
```
xcode-project build-ipa \
              --workspace Runner.xcworkspace \
              --scheme Runner \
              --config Flutter/Release.xcconfig 
```
